packages/jest-cli/src/lib/update_global_config.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+import type {GlobalConfig, SnapshotUpdateState} from 'types/Config';
+
+type Options = {
+  testNamePattern?: string,
+  testPathPattern?: string,
+  noSCM?: boolean,
+  updateSnapshot?: SnapshotUpdateState,
+};
+
+module.exports = ({
+  globalConfig,
+  mode,
+  options,
+}: {
```

Comment:
actually you're right, i don't see any reason to keep it two separate arguments. i'll just merge them into
```js
updateGlobalConfig(globalConfig: GlobalConfig, options: Options);
```